src/app/dashboard/RegisterUser.js

```diff
@@ -65,9 +67,12 @@ export default function RegisterUser() {
       });
       let newUser = {
         ...tenant,
+        ...state,
+        lastlogin: "Mar 01 2022 11:51:39",
         password: tenant.email.substring(0, tenant.email.search("@")),
```

Comment:
avoid using password in the frontend, unecessary as it is not used

Reply:
It is a temporary generated password for tenant,It will be removed onces Api integration will done.